src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -91,7 +94,56 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
     componentDidMount(){
 
+    }
+
+    componentDidUpdate() {
+        this.addBookmarkButtons();
+    }
+
+    public addBookmarkButtons() {
+        let shareURL = window.location.href;
+
+        if (shareURL.indexOf("session_id") < -1) return;
```

Comment:
you can now use string.includes or /session_id/.test(str) which are boolean

Reply:
Fixed.